jtalks-common-model/src/main/resources/org/jtalks/common/migrations/V8__New_Entities.sql

```diff
@@ -58,7 +58,7 @@ CREATE TABLE `GROUP_USER_REF` (
 
 CREATE TABLE `PROPERTIES` (
   `PROPERTY_ID` bigint(20) NOT NULL AUTO_INCREMENT,
-  `UUID` varchar(255) NOT NULL,
+  `UUID` varchar(100) NOT NULL,
   `NAME` varchar(255) NOT NULL,
```

Comment:
What's so different between UUID and NAME? Why UUID is decreased while NAME is left unchanged?